cars.js

```diff
@@ -100,12 +105,37 @@ function getListItem(carData) {
 }
 
 function getCardItem(carData) {
+
     const card = document.createElement("div");
-    card.style.border = "1px solid black";
-    card.style.height = "50px";
-    card.style.width = "300px";
-    card.style.display = "inline-block";
-    card.innerText = `car lp: ${carData.lp}, car color: ${carData.color}`;
+    card.classList.add("card") 
+    card.style = "width: 18rem; margin: 10px"
```

Comment:
use class 
no style
card.style = "width: 18rem; margin: 10px"

Reply:
the width is how its written in Bootstrap , and the margin i added to make it look more prettier